assets/app/scripts/filters/resources.js

```diff
@@ -801,4 +801,46 @@ angular.module('openshiftConsole')
       };
       return nameFormatMap[resourceType] || resourceType;
     };
+  })
+  .filter('routeTargetPortMapping', function(RoutesService) {
+    var portDisplayValue = function(servicePort, containerPort, protocol) {
+      servicePort = servicePort || "<unknown>";
+      containerPort = containerPort || "<unknown>";
+      var mapping = "Service Port " + servicePort  + " \u2192 Container Port " + containerPort;
```

Comment:
Tried to explain it in the comment just below on the filter, but I guess not clear. I can add one here too